docs/退费异常类工单/【保险】退费异常订单线下退款线上闭环能力.md

```
@@ -0,0 +1,56 @@
+2. ~~退费行为改为异步滞后执行，不影响保司侧状态更新。~~
+
+~~（原流程：退费成功后才通知保司，导致异常时保司状态长期挂起）~~
+
+1. 退保失败直接人工跟进
+  1. 若保司退保接口返回失败，系统记录失败原因，转入人工处理流程，不再盲目重试。
+2. 退费异常告警后分流处理
+  1. **退费异常为卡异常：**通知支付网关中断该订单重试，并调用异常告警接口（由工单系统提供，以实际命名接口为准），将退费卡异常数据同步至客服工单系统。**接口入参**含：保单号、内部订单号、退费失败原因、应退金额、申请时间等（以实际的为准）
+  2. **退费异常为非卡异常：**系统正常启动 48小时重试策略，若12小时仍未处理成功，则：告知支付网关中断重试（避免无效扣费）；保险系统调用异常告警接口（由工单系统提供，以实际命名接口为准），将退费异常非卡异常数据同步至客服工单系统。**接口入参**含：保单号、内部订单号、退费失败原因、应退金额、申请时间等（以实际的为准）
+3. 工单系统自动生成“线下退费”工单@李乾行，**其中页面原型如下所示（具体内容以实际为准）：**
+  1. 客服工单系统接收异常数据后，自动创建类别为**卡异常退费失败；其它异常退费失败**的工单**。**该类别工单详情新增退费模块，模块内容含**异常原因、保单号、内部订单号、应退金额、实退金额，补偿金、申请时间（以实际为准）。**
+  2. 其中异常原因、保单号、期次、应扣金额由保险异常告警接口推给工单系统展示，**本着尽量不操作原则，客服仅需要在必要情况下填写补偿金金额，其余应退，实退金额都是按接口推数内容展示。之后，推数内容**由工单系统接口返回给保险系统，**接口出参**含异常原因、保单号、内部订单号、应退金额、实退金额，补偿金、已处理时间（以实际为准）。
```

markdownlint-cli2 (0.23.2)

[warning] Inconsistent indentation for list items at the same level
Expected: 0; Actual: 2

(MD005, list-indent)